### ISSUE TITLE
UI: back button consistency

### ISSUE THREAD

**Issue by durkie** [PublicDesignWorkshop/FoodParent#8]
I know there are issues around single-page apps and the back button, but we should have consistent styling:

![image](https://cloud.githubusercontent.com/assets/3200608/13996392/d132b8f8-f103-11e5-932e-a10ebd044f6e.png)
(back/close is an "X" in upper right)

![image](https://cloud.githubusercontent.com/assets/3200608/13996400/e2c5105c-f103-11e5-97ea-5fce8aafd18d.png)
(back/close is a button in upper left)

![image](https://cloud.githubusercontent.com/assets/3200608/13996416/fba8584a-f103-11e5-91b6-20e03b26411f.png)
(back/close is an "X" in upper right)

**Comment by ThePlushEater** [comment]
I agree. I will talk with June and try to find the better solution for that. Thank you for a good feedback.
